entries/e/ee.co.uk.json

```diff
@@ -1,10 +1,12 @@
 {
   "EE": {
     "domain": "ee.co.uk",
-    "contact": {
-      "facebook": "ee",
-      "twitter": "EE"
-    },
+    "url": "https://www.ee.co.uk/",
```

Comment:
Thanks for the pull request @dalDevelo!
Could you please revert this change?
I'm not sure why the build failed earlier, but it seems to have been due to some random unrelated event, as the URL element shouldn't affect the domain rank.
```suggestion
```